bugout/journal.py

```diff
@@ -352,6 +381,7 @@ def search(
         token: Union[str, uuid.UUID],
         journal_id: Union[str, uuid.UUID],
         query: str,
+        filters: List[str] = [],
```

Comment:
Not good to have mutable default value (e.g. `[]`)

Right pattern is:
```
filters: Optional[List[str]] = None
```

and then, in function,
```
if filters is None:
    filters = []
```